prisma/schema.sql

```diff
@@ -226,7 +227,7 @@ CREATE TABLE application_processing (
   review_request_completed_employee_id INTEGER,
   review_request_completed_updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
   invoice_number INTEGER UNIQUE,
-  documents_url VARCHAR(255),
+  documents_url TEXT,
```

Comment:
added URL here so that we don't have to generate it every time we are displaying it. There's a backend check when querying to keep the temporary url fresh

Reply:
Are URLs not guaranteed to fit in 255 chars?

Reply:
yeah ran into this while testing. Presigned urls have lots of query params which causes it to go over